datajoint/table.py

```diff
@@ -683,8 +683,9 @@ def __init__(self, arg, database=None):
         self.database = database
         self._connection = arg
         self._definition = """    # event logging table for `{database}`
-        timestamp = CURRENT_TIMESTAMP : timestamp
+        id :  smallint auto_increment
```

Comment:
`smallint` may not be enough.

Reply:
Can we keep the timestamp primary key with the higher precision or is collision probability still high?

Reply:
Is there a reason why we shouldn't use an incremental primary key here? I would expect this to be the standard to prevent logging collisions. As far as an example use case: On our flask web deployments, I could see possible collisions near the millisecond level as the community grows. In the past I've used `bigint` for tables similar to this; just wasn't sure if we wanted to go there yet.

Reply:
yes, it's fine to go with autoincrement or `serial` datatype. There is not that much logging going on in datajoint, but I agree that we may as well avoid any possibility of collisions.

Reply:
Had not come across `serial` but it looks ideal here, thanks! I have now updated the changes to leverage it.